src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -30,17 +36,46 @@ class ReviewCollapsibleChapter extends React.Component {
     this.handleEdit = this.handleEdit.bind(this);
   }
 
-  /* eslint-disable-next-line camelcase */
-  UNSAFE_componentWillMount() {
-    this.id = uniqueId();
-  }
```

Comment:
Did bit of code doesn't appear to be used anywhere, so I thought it'd be best to remove it